sympy/functions/combinatorial/numbers.py

```diff
@@ -457,13 +457,23 @@ class bernoulli(Function):
     * ``bernoulli(n)`` gives the nth Bernoulli number, `B_n`
     * ``bernoulli(n, x)`` gives the nth Bernoulli polynomial in `x`, `B_n(x)`
 
+    .. versionchanged:: 1.11
+        ``bernoulli(1)`` gives `+\frac12` instead of `-\frac12`.
```

Comment:
```suggestion
        ``bernoulli(1)`` gives `+\frac{1}{2}` instead of `-\frac{1}{2}`.
```
Sure, the current works for this particular case, but I think this is much clearer in general.

Reply:
(NB: Stripping all not-strictly-needed braces and such is the way I write LaTeX in my Mathematics Stack Exchange answers.)

Reply:
For `\frac` the braces help make it a little more human readable, which matters for docstrings since many people will read them there.